include/bactria/Category.hpp

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2021 Jan Stephan
+ *
+ * Licensed under the EUPL, Version 1.2 or - as soon they will be approved by
+ * the European Commission - subsequent versions of the EUPL (the “Licence”).
+ * You may not use this work except in compliance with the Licence.
+ * You may obtain a copy of the Licence at:
+ *
+ *     http://ec.europa.eu/idabc/eupl.html
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the Licence is distributed on an “AS IS” basis, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  Licence permissions and limitations under the Licence.
```

Comment:
I am a big fan of SPDX license identifiers to get rid of these wordy headers.

```suggestion
 * SPDX-License-Identifier: EUPL-1.2
```

See also: https://spdx.org/licenses/EUPL-1.2.html

Even more and against popular practice, I try to avoid license headers including copyright lists. The repository itself already has a license and a git history. And the age of people emailing around files is hopefully over ;) But this is just my personal stance.

Reply:
The copyright list is mandated by our coding style IIRC.